less/plugins/adapt-contrib-gmcq/gmcq.less

```diff
@@ -1,160 +1,71 @@
-            }
-            .gmcq-item-inner {
-                color: @item-color-disabled;
-            }
-        }
-    }
-
-    .gmcq-item-inner {
-        margin-left: @icon-size * 2;
-        color: @item-color;
-        padding-bottom: @item-padding-bottom;
-        padding-top: @item-padding-top;
-        padding-left: 0;
-        padding-right: 0;
-        .dir-rtl & {
-            margin-left: inherit;
-            margin-right: @icon-size * 2;
-        }
-    }
-
-    .gmcq-item input {
-        filter: alpha(opacity=0); // ie8 fix for opacity on input
-        opacity: 0;
-    }
-
-    .gmcq-correct-icon {
-        color: @validation-success;
-    }
-
-    .gmcq-incorrect-icon {
-        color: @validation-error;
-    }
-
-    .gmcq-widget {
-        &.show-user-answer {
-            label.selected {
-                .gmcq-item-inner {
-                    color: @item-color-selected;
-                }
-            }
-        }
-
-        &.show-correct-answer {
-            label.selected {
-                .gmcq-item-inner {
-                    color: @item-color-selected;
-                }
-            }
-        }
-
-        &.disabled {
-            .gmcq-item label.selected.disabled {
-                background-color: transparent;
-                .gmcq-item-icon {
-                    &.gmcq-correct-icon {
-                        color:@validation-success;
-                    }
-                    &.gmcq-incorrect-icon {
-                        color:@validation-error;
-                    }
-                }
-            }
-
-            .gmcq-item label.disabled {
-                color: @item-color-disabled;
-                background-color: transparent;
-                border-color: @item-border-color-disabled;
-                border: none;
-            }
-        }
-    }
+.gmcq {
+  &__item-label {
+    margin: @item-margin;
+  }
+
+  // -------------------------------------------------
+  // Layout #1
+  // --------------------------------------------------
+  &__item-option {
+    margin-top: @item-margin;
+    padding: @item-padding / 2;
+    background-color: @item-color;
+    color: @item-color-inverted;
+    border-radius: 5px;
+  }
+
+  .no-touch &__item-label:not(.is-disabled):not(.is-selected):hover .gmcq__item-option {
+    background-color: @item-color-hover;
+    color: @item-color-inverted-hover;
+    .transition(background-color @duration ease-in, color @duration ease-in;);
+  }
+
+  &__item-label.is-disabled .gmcq__item-option {
+    background-color: @disabled;
+    color: @disabled-inverted;
+  }
+
+  &__item-label.is-selected .gmcq__item-option {
+    background-color: @item-color-selected;
+    color: @item-color-inverted-selected;
+  }
+
+  &__item-state {
+    min-height: @icon-size;
+  }
+
+  &__item-text {
+    margin-top: @item-margin;
+    text-align: center;
+  }
+  // --------------------------------------------------
+
+  // --------------------------------------------------
```

Comment:
Should these be deleted?

Reply:
I recommend leaving comment in as code allows different visual style for gmcq items.